src/commands/run-tests.yml

```diff
@@ -14,6 +14,12 @@ parameters:
       folders using a colon-separated or semicolon-separated list.
     type: string
     default: ''
+  select-by-name:
+    description: >
+      Name or pattern used to select test suite elements. To create a test suite, MATLAB uses only the test elements
+      that match the specified name or pattern.
+    type: string
+    default: ''
   select-by-folder:
     description: >
```

Comment:
Please update the second phrase like this:  

, specified as a path relative to the project root folder.

Reply:
Updated the description